contribs/discrete_mode_choice/src/test/java/org/matsim/contrib/discrete_mode_choice/examples/TestSiouxFalls.java

```diff
@@ -56,9 +56,9 @@ public void install() {
 
 		controller.run();
 
-		assertEquals(44196, (int) listener.counts.get("pt"));
+		assertEquals(44195, (int) listener.counts.get("pt"));
 		assertEquals(132316, (int) listener.counts.get("car"));
-		assertEquals(82140, (int) listener.counts.get("walk"));
+		assertEquals(82139, (int) listener.counts.get("walk"));
```

Comment:
If you rebase on the current master branch, it should be resolved (maybe not in the nicest way though).

Reply:
I did a merge from master an manually took the lines from the master.